spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,137 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+  describe 'GET users#index' do
+
+    before do
+      create(:user)
```

Comment:
You can use a loop to create the users you want to.

Reply:
Done with create_list (FactoryBot method)